### PR TITLE
Minor documentation improvement

### PR DESCRIPTION
**Pull request by fullonic**
refs #85

Please provide enough information so that others can review your pull request:
Improves documentation about, "how to use `pre-commit`" locally. Also informs that users with permission can create PR directly against this repository.

### CODE REVIEW

**Comment by fullonic** [comment]
Once #84 is merged conflicts will be solved.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,7 +6,10 @@ There are many ways in which you can contribute, beyond writing code. The goal o
 
 ## How to contribute
 
-1. Fork this very repository (use the `fork` button at the top left)
+1. Fork this very repository (use the `fork` button at the top left).
+
+**Tip**: If you have permission, use the [GeeksCAT fork](https://github.com/GeeksCAT/anem-per-feina) directly instead of create a new fork.
```
Comment (fullonic) [inline CONTRIBUTING.md:11]: This **Tip** here is understandable?

Comment (XaviTorello) [reply CONTRIBUTING.md:11]: I think that it should be something like: `**Tip for hacktoberfest'20 participants**: If you have doubts or troubles forking and keeping it synchronized, start a new `branch` from `master` and submit a PR directly in this repo.`

**Review by XaviTorello — CHANGES_REQUESTED** [review]
Great! Manu thanks @fullonic :)

Just an idea regarding the `fork vs push` tip

File: CONTRIBUTING.md (hunk 1 above)
Comment (fullonic) [reply CONTRIBUTING.md]: Great! It's more explicit, and explicit is better than implicit :)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -8,11 +8,11 @@ There are many ways in which you can contribute, beyond writing code. The goal o
 
 1. Fork this very repository (use the `fork` button at the top left).
 
-**Tip**: If you have permission, use the [GeeksCAT fork](https://github.com/GeeksCAT/anem-per-feina) directly instead of create a new fork.
+**Tip for hacktoberfest'20 participants**: If you have doubts or troubles forking and keeping it synchronized, start a new branch from **master** and submit a PR directly in this  [repo](https://github.com/GeeksCAT/anem-per-feina).
```
Comment (XaviTorello) [inline CONTRIBUTING.md:11]: typo: double space in `this  [repo](`

**Review by XaviTorello — CHANGES_REQUESTED** [review]
Just fix this typo and will be merged :)

**Review by XaviTorello — APPROVED** [review]
Great! Many thanks @fullonic :beers:

**Pull request merged**